client/app/stores/commentsStore.js

```diff
@@ -5,11 +5,10 @@ import reducers from '../reducers';
 import { initalStates } from '../reducers';
 
 export default props => {
-  const initialComments = props;
   const { $$commentsState } = initalStates;
   const initialState = {
     $$commentsStore: $$commentsState.merge({
-      $$comments: initialComments,
+      $$comments: props,
```

Comment:
@dylangrafmyre yep, we just need to put `/* eslint react/prop-types: 0 */` at the top of the file, see bc076958012b176fa5aee1f5347e5153400b2426.